packages/db/src/types.ts

```diff
@@ -87,9 +87,11 @@ export type TNewLessonCompletion = typeof schema.lessonCompletion.$inferInsert;
 
 export type TCommunityAnswer = typeof schema.communityAnswer.$inferSelect;
 export type TNewCommunityAnswer = typeof schema.communityAnswer.$inferInsert;
+export type UpVoteAnswerParams = Pick<TNewCommunityAnswer, 'id' | 'votes'>;
 
 export type TCommunityQuestion = typeof schema.communityQuestion.$inferSelect;
 export type TNewCommunityQuestion = typeof schema.communityQuestion.$inferInsert;
+export type UpVoteQuestionParams = Pick<TNewCommunityQuestion, 'id' | 'votes'>;
```

Comment:
You shouldn't make any change to this file. It is auto generated.

Create the type in the place it is needed.

This file is only for db tables

Reply:
fixed